examples/dashboard/dashboard.py

```diff
@@ -28,7 +31,6 @@
 from webargs import fields
```

Comment:
It's just an example, so my opinion is that the dependencies don't matter as long as their listed explicitly somewhere.